packages/commonwealth/server/routes/createComment.ts

```diff
@@ -190,6 +191,14 @@ const createComment = async (
     // check always passes if the comment text isn't a Quill document
   }
 
+  await verifyComment(canvas_action, canvas_session, canvas_hash, {
```

Comment:
Under what circumstances might verification fail? Is there any reason to be worried about users accidentally being unable to post, e.g. if they're magic link users and can't sign messages?

Reply:
Wallet holding accounts should always be able to post, given that the logic flow generates a signing key for every login method we use. This covers all mainline chains (Eth, Cosmos, Polkadot, Solana, Cosmos variants) but we should test those.

- Near logins generate a signing key, but the signature check isn't fully enforced because NEAR addresses are human-readable names stored in a registry on their chain, not hashed pubkeys.
- Axie/Ronin logins behave similarly.

Can't test Near or Axie, per my message in Slack.